### ISSUE TITLE
Examples recommend using chaiAsPromised but it has problems with some promises.

### ISSUE THREAD

**Issue by envygeeks** [electron/spectron#71]
chaiAsPromised doesn't properly handle rejections, and constantly reports true for some `spectron` methods when using some Promised methods.  For example:

```js
it("should be true", function() {
  let val = this.app.electron.remote.getGlobal("appConfig").then(v => { return v.val })
  assert.eventually.equal(
    val, true
  )
})
```

Always returns true for us, and never actually causes an assertion failure because there was a rejection with the object having an error.  We could catch it, but then what is the point of chaiAsPromised at that point?

---

I propose the documentation also recommend a better way to deal with promises if they are using ES7.
```js
it("should be true", async function() {
  let config = await this.app.electron.remote.getGlobal("appConfig")
  assert.equal(
    config.val, true
  )
})
```

If this is acceptable, I'm happy to send a pull request.

**Comment by kevinsawicki** [comment]
> If this is acceptable, I'm happy to send a pull request.

Yeah, documentation updates are welcomed and appreciated, thanks.
